apps/src/craft/code-connection/cc-client.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * CC client for 3rd party extension
+ *
+ */
+
+export default class cc_client {
+    constructor(port) {
+        this.port = port;
+        this.baseUrl = "http://localhost:";
```

Comment:
Could make this a local const given that it looks like we dont expect it to ever change.